config/prod/synapse-login-scipoolprod-params.yaml

```diff
@@ -1,5 +1,8 @@
 template_path: ssm-parameters.j2
```

Comment:
Shouldn't this be `remote/ssm-parameters.j2` ?

Reply:
fixed